src/migrations/20220316143724_payment_gateway_meta_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class RecurrentPaymentStateIndex extends AbstractMigration
+{
+    public function up()
+    {
+        if ($this->hasTable('payment_gateway_meta')) {
+            return;
+        }
+
+        $this->table('payment_gateway_meta')
+            ->addColumn('payment_gateway_id', 'integer', ['null' => false])
+            ->addColumn('key', 'string', ['null' => false, 'limit' => 255])
+            ->addColumn('value', 'string', ['null' => false, 'limit' => 255])
+            ->addForeignKey('payment_gateway_id', 'payment_gateways', 'id', ['delete' => 'CASCADE', 'update' => 'CASCADE'])
+            ->save();
```

Comment:
Internally we have an agreement not to use `save()` and be explicit with our intention. If we know we're creating the table, we use `create()`.